File: Storage/Code/326.3-的幂.cpp
/*
 * @lc app=leetcode.cn id=326 lang=cpp
 *
 * [326] 3的幂
 */

// @lc code=start
class Solution {
public:
    bool isPowerOfThree(int n) {
        if(n<=0)    return false;
        if(n==1)    return true;
        while(n!=1){
            if(n%3!=0)  return false;
            n/=3;
        }
        return true;
    }
};
// @lc code=end

File: Storage/Code/326.3-的幂.py
#
# @lc app=leetcode.cn id=326 lang=python3
#
# [326] 3的幂
#

# @lc code=start
class Solution:
    def isPowerOfThree(self, n: int) -> bool:
        if n<=0:
            return False
        if n==1:
            return True
        while n!=1:
            if n%3!=0:
                return False
            n=n/3
        return True
# @lc code=end

